web/src/main/java/org/cbioportal/web/parameter/ChartType.java

```diff
@@ -0,0 +1,12 @@
+package org.cbioportal.web.parameter;
+
+public enum ChartType {
```

Comment:
This is just too highly coupled with frontend and would be tricky to add more chart type in the future. Backend really does not care what chart type is. Can we just make the chart type string?

Reply:
Add this would enforce type system. This would be very useful while testing and avoids any errors. and in future if we want to send default configs through api, this would be useful.

Reply:
would be good to have a comment that points to the frontend file where these types are defined